apps/files/src/components/Collaborators/Collaborator.vue

```diff
@@ -3,9 +3,35 @@
     <oc-table-row v-if="$_reshareInformation" class="files-collaborators-collaborator-table-row-top">
       <oc-table-cell shrink :colspan="firstColumn ? 2 : 1"></oc-table-cell>
       <oc-table-cell colspan="2">
-        <div class="uk-text-meta uk-flex uk-flex-middle">
-          <oc-icon name="repeat" class="uk-preserve-width" />
-          <span class="uk-padding-remove uk-margin-xsmall-left uk-text-truncate files-collaborators-collaborator-reshare-information">{{ $_reshareInformation }}</span>
+        <div class="uk-text-meta">
+          <oc-button type="a" variation="raw" :id="$_resharerToggleId" :aria-label="$gettext('Show resharer details')">
```

Comment:
Without the `type` "a" pls. It's important to keep the button in here for semantics and accessibility.

Reply:
why keep the type attribute at all then ? should we kill it if we cannot use it ?

Reply:
adjusted and retested, it looks the same